File: Beakjun_online_judge/1934.py
import sys

def gcd(a, b):
    if b == 0:
        return a
    else:
        return gcd(b, a%b)

def lcm(a, b, g):
    return (int) (a*b/g)

def main():
    testcase = int(sys.stdin.readline())

    for i in range(0, testcase):
        input = list(map(int, sys.stdin.readline().split()))
        
        a = input[0]
        b = input[1]

        g = gcd(a, b)
        l = lcm(a, b, g)
        print(l)

if __name__ == "__main__":
    main()

File: Beakjun_online_judge/1934.cpp
#include <iostream>
using namespace std;

int gcd(int a, int b){
    if(b == 0) return a;
    else return gcd(b, a%b);
}

int lcm(int a, int b, int g){
    return a*b/g;
}

int main(){
    int test_case, a, b, g, l;
    cin >> test_case;

    for(int i=0; i<test_case; i++){
        cin >> a >> b;
        g = gcd(a, b);
        l = lcm(a, b, g);
        cout << l << endl;
    }
    
    return 0;
}